integration-tests/braintree_test.go

```diff
@@ -31,30 +54,154 @@ func TestBraintree(t *testing.T) {
 	}
 }
 
-func TestBraintreeFailedAuth(t *testing.T) {
+// TestBraintreeAuthCapture
+//
+// Tests a Braintree authorization then full capture
+func TestBraintreeAuthCapture(t *testing.T) {
 	client := braintree.NewClient(&braintree.Credentials{
 		MerchantID: getEnv("BRAINTREE_MERCHANT_ID"),
 		PublicKey:  getEnv("BRAINTREE_PUBLIC_KEY"),
 		PrivateKey: getEnv("BRAINTREE_PRIVATE_KEY"),
 	})
 	authRequest := sleet_testing.BaseAuthorizationRequest()
-	authRequest.Amount = sleet.Amount{
-		Amount:   201000,
-		Currency: "USD",
+	auth, err := client.Authorize(authRequest)
+	if err != nil {
+		t.Errorf("Expected no error: received: %s", err)
 	}
-	authRequest.BillingAddress = &sleet.BillingAddress{
-		StreetAddress1: sPtr("77 Geary St"),
-		StreetAddress2: sPtr("Floor 4"),
-		Locality:       sPtr("San Francisco"),
-		RegionCode:     sPtr("CA"),
-		PostalCode:     sPtr("94108"),
-		CountryCode:    sPtr("US"),
+	if !auth.Success {
+		t.Errorf("Expected Success: received: %s", auth.ErrorCode)
 	}
-	resp, err := client.Authorize(authRequest)
+
+	captureRequest := &sleet.CaptureRequest{
+		Amount:               &authRequest.Amount,
+		TransactionReference: auth.TransactionReference,
+	}
+	capture, err := client.Capture(captureRequest)
+	if err != nil {
+		t.Error("Capture request should not have failed")
+	}
+
+	if capture.Success == false {
```

Comment:
Nit: !capture.Success (same for the 6 below)

Reply:
Wait, this says if `capture.Success is false` throw error. Seems correct to me @jakechiang-bolt

Reply:
Oh I just meant it as a stylistic change, switching `if capture.Success == false` to `if !capture.Success`.

Reply:
Oh yeah nice catch! I'll change them

Reply:
Oh I see, haha, ya sounds good